components/automate-ui/src/app/app-routing.module.ts

```diff
@@ -102,6 +103,10 @@ const routes: Routes = [
           path: 'tokens/:id',
           component: ApiTokenDetailsComponent
         },
+        {
+          path: 'sso',
+          component: ConfigureComponent
```

Comment:
Rename the component from ConfigureComponent to SSOComponent

Reply:
resolved